cargo-fix/tests/all/vcs.rs

```diff
@@ -0,0 +1,75 @@
+use super::project;
+
+#[test]
+fn warns_if_no_vcs_detected() {
+    let p = project()
+        .use_temp_dir()
+        .file(
+            "src/lib.rs",
+            r#"
+            pub fn foo() {
+            }
+        "#,
+        )
+        .build();
+
+    p.expect_cmd("cargo-fix fix")
+        .check_vcs(true)
+        .stderr(
+            "warning: Could not detect a version control system\n\
+             You should consider using a VCS so you can easily see and revert rustfix' changes.\n\
```

Comment:
missing _s_ (`rustfix's`)?

Reply:
`rustfix's` looks super weird… [but you are right](https://ell.stackexchange.com/questions/6295/when-a-word-ends-in-s-or-x-do-you-add-s-or-just-an) -- PR welcome!